Userland/Libraries/LibJS/Runtime/StringConstructor.cpp

```diff
@@ -120,4 +121,26 @@ JS_DEFINE_NATIVE_FUNCTION(StringConstructor::from_char_code)
     return js_string(vm, builder.build());
 }
 
+// 22.1.2.2 String.fromCodePoint ( ...codePoints ), https://tc39.es/ecma262/#sec-string.fromcodepoint
+JS_DEFINE_NATIVE_FUNCTION(StringConstructor::from_code_point)
+{
+    StringBuilder builder;
+    for (size_t i = 0; i < vm.argument_count(); ++i) {
+        auto next_code_point = vm.argument(i).to_number(global_object);
+        if (vm.exception())
+            return {};
+        if (!next_code_point.is_integer()) {
```

Comment:
Side note, `is_integer()` seems to match https://tc39.es/ecma262/#sec-isintegralnumber and should be renamed accordingly, I think :)

Reply:
Oh yeah, I've just been using it as if its equivalent, do you want to change it as part of this PR or another one? (or do you wanna do it?)

Reply:
Just thinking out loud really :D
Feel free to rename it separately, up to you!